src/ref.builds

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\ref\**\*.*proj">
+    <Project Include="*\ref\**\*.builds;*\ref\**\*.*proj">
```

Comment:
I do think adding a .builds file under the ref folder is the most consistent thing for us to do.